src/app/analytics/analytics.factory.js

```diff
@@ -555,6 +555,7 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, envService
 
         // Set the transactionId in localStorage to be the one that is passed in
         sessionStorage.setItem('transactionId', currentTransactionId)
+        const jobId = new URLSearchParams($location.search()).get('utm_term')
```

Comment:
Any sort of fallback value for this? Most of the time, `utm_term` won't be in the URL.

Reply:
It will be `undefined` if it is missing. At least when I was working with the Adswerve people on branded checkout, they said to set unavailable attributes to `undefined`.

Reply:
According to [these docs](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) it will return `null` rather than `undefined` when calling `get('missing_parameter)`

Reply:
That's correct, just confirmed it in a window

Reply:
You're right. That's fixed now.

Reply:
Looks like it was fixed on branded, but not standard checkout analytics file.